test-scripts/tomcat-deployment/cf-templates/add-instances.yaml

```diff
@@ -0,0 +1,73 @@
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year should be 2018

Reply:
Fixed with 1479a08a77d83ce3fe12f93a1f691490f7e8e860